### ISSUE TITLE
Bottom of app has tiny white line

### ISSUE THREAD

**Issue by PanderMusubi** [CNugteren/NLWeer#7]
Bottom of app has tiny white line which should not be there, see attachment.

![Screenshot_20191215-083517_Trebuchet](https://user-images.githubusercontent.com/198964/70860882-8728b900-1f27-11ea-8c7d-82b003750800.png)

**Comment by CNugteren** [comment]
This does not show on my device. Could you share your screen dimensions so I can try it on the Android emulator?

**Comment by PanderMusubi** [comment]
2160x1080 OnePlus 5T with LineageOS

**Comment by CNugteren** [comment]
I tried to reproduce it with a virtual device with the same resolution (Android Pie API 29), and I managed to see the white line, but only in the app-switcher, not when using the app full screen:

![nlweer_full_screen](https://user-images.githubusercontent.com/1162108/70986432-2aabd200-20be-11ea-9566-a3afd89bdac0.png)
![nlweer_app_switcher](https://user-images.githubusercontent.com/1162108/70986433-2aabd200-20be-11ea-9891-708cb7412c59.png)

Is that also what you are observing? In that case it is really a minor issue: I spent a bit of time trying to fix it but couldn't easily find the bug. So I propose to leave it at this unless someone else one wants to have a go. What do you think?

**Comment by PanderMusubi** [comment]
Yes, only when task switching. It is a minor bug indeed but caught my eye as no other apps did that.

**Comment by PanderMusubi** [comment]
Also fixed for me.
